generators/client/templates/angular/src/test/javascript/spec/app/shared/alert/alert-error.component.spec.ts.ejs

```diff
@@ -0,0 +1,137 @@
+import { ComponentFixture, TestBed, async, inject, fakeAsync, tick } from '@angular/core/testing';
+import { HttpErrorResponse, HttpHeaders } from '@angular/common/http';
+import { JhiAlertService, JhiEventManager } from 'ng-jhipster';
+<%_ if (enableTranslation) { _%>
+import { TranslateModule } from '@ngx-translate/core';
+<%_ } _%>
+
+import { <%=angularXAppName%>TestModule } from '../../../test.module';
+import { <%=jhiPrefixCapitalized%>AlertErrorComponent } from 'app/shared/alert/alert-error.component';
+import { MockAlertService } from "../../../helpers/mock-alert.service";
+
+describe('Component Tests', () => {
+    describe('Alert Error Component', () => {
+        let comp: <%=jhiPrefixCapitalized%>AlertErrorComponent;
+        let fixture: ComponentFixture<<%=jhiPrefixCapitalized%>AlertErrorComponent>;
+        let eventManager: JhiEventManager;
+
+        beforeEach(async(() => {
+            TestBed.configureTestingModule({
+                imports: [<%=angularXAppName%>TestModule<% if (enableTranslation) { %>, TranslateModule.forRoot()<% } %>],
+                declarations: [<%=jhiPrefixCapitalized%>AlertErrorComponent],
+                providers: [
+                    JhiEventManager,
+                    {
+                        provide: JhiAlertService,
```

Comment:
do we require prefix variable here?

Reply:
`JhiEventManager` and `JhiAlertService` come from `ng-jhipster` so it always uses the `jhi` prefix